task-sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -1034,7 +1034,7 @@ def _handle_request(self, msg: ToSupervisor, log: FilteringBoundLogger):
             var = self.client.variables.get(msg.key)
             if isinstance(var, VariableResponse):
                 if var.value:
-                    mask_secret(var.value)
+                    mask_secret(var.value, var.key)
```

Comment:
Tests please!

Reply:
Yeah just pushed one through https://github.com/apache/airflow/pull/50775/commits/5ec9a8d6d1ccf05c8156eefab8a661a6ab4cca92.


Wondering if i should have it be more generic. WDYT?